pkg/sdn/plugin/ovscontroller.go

```diff
@@ -407,6 +407,7 @@ func (oc *ovsController) UpdateEgressNetworkPolicyRules(policies []osapi.EgressN
 		otx.AddFlow("table=100, reg0=%d, cookie=1, priority=65535, actions=drop", vnid)
 		otx.DeleteFlows("table=100, reg0=%d, cookie=0/1", vnid)
 
+		otx.AddFlow("table=100, reg0=%d, priority=%d, ip, nw_dst=%s actions=output:2", vnid, len(policies[0].Spec.Egress)+1, nodeIP)
```

Comment:
You don't need a separate rule for each VNID. Just have a single rule (created in SetupOVS) with no reg0 match, with higher priority than any flow created by UpdateEgressNetworkPolicyRules. (An EgressNetworkPolicy can only have `osapi.EgressNetworkPolicyMaxRules` rules in it, so that plus 1 is high enough priority.)